src/sql/base/browser/ui/table/formatters.ts

```diff
@@ -119,13 +119,18 @@ export function textFormatter(row: number | undefined, cell: any | undefined, va
 				cellStyle = value.style;
 			}
 		}
-		valueToDisplay = escape(valueToDisplay.length > 250 ? valueToDisplay.slice(0, 250) + '...' : valueToDisplay);
+		valueToDisplay = getCellDisplayValue(valueToDisplay);
 		titleValue = valueToDisplay;
 	}
 
 	return `<span title="${titleValue}" style="${cellStyle}" class="${cellClasses}">${valueToDisplay}</span>`;
 }
 
+function getCellDisplayValue(cellValue: string): string {
+	let valueToDisplay = cellValue.replace(/(\r\n|\n|\r)/g, ' ');
```

Comment:
I'm probably missing something but this appears to replace the newlines with a space.  Where are the visible linebreak characters in the screenshot coming from?

Reply:
good catch, I forgot to push the last commit. done now.